src/OrleansRuntime/GrainDirectory/MultiClusterRegistration/GlobalSingleInstanceRegistrar.cs

```diff
@@ -95,29 +95,27 @@ public void Delete(GrainId gid)
                 if (logger.IsVerbose)
                     logger.Verbose("GSIP:Req {0} Round={1} Act={2}", address.Grain.ToString(), numRetries - retries, myActivation.Address.ToString());
 
-                var responses = SendRequestRound(address, remoteClusters);
-
-                var outcome = await responses.Task;
+                var outcome = await SendRequestRound(address, remoteClusters);
 
                 if (logger.IsVerbose)
-                    logger.Verbose("GSIP:End {0} Round={1} Outcome={2}", address.Grain.ToString(), numRetries - retries, responses);
+                    logger.Verbose("GSIP:End {0} Round={1} Outcome={2}", address.Grain.ToString(), numRetries - retries, outcome.State);
```

Comment:
no, I did not, I thought the ToString was mostly to check whether the tracking completed or not (and at this point after the await it was always completed), and I missed that. So I'll re-add this information in the `GlobalSingleInstanceResponseOutcome` class as you suggested in your other comment